### PR TITLE
Documentation update, added info about the cloud storage UI and add AWS-S3 tutorial

### PR DESCRIPTION
**Pull request by TOsmanov**
### Motivation and context

Depended #3724

Issues:
- To generate a manifest file, you need to download all the files to your pc that will be inconvenient for datasets of large volume. As well, adding data to the dataset will require updating the manifest file.
   It may be worth adding support for [S3 inventory](https://docs.aws.amazon.com/AmazonS3/latest/userguide/storage-inventory.html) in CVAT. In the S3 inventory contains paths to files with extension, and you can also add an additional field `ETag` which can be an MD5 digest of the object data.
-  After the file manifest is created, the `name` parameter has `\\` Instead of `/`.  For fix this, I used `.replace` in `core.py` file, perhaps there is a better solution.

Preview:
https://tosmanov.github.io/cvat/docs/manual/basics/attach-cloud-storage/

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: site/content/en/docs/manual/basics/attach-cloud-storage.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+You can also add tags for the user (optional), and look again at the entered data. In the last step of creating a user,
+you will be provided with `access key ID` and `secret access key`,
+they will need to be used in CVAT when adding cloud storage.
+
+![](/images/aws-s3_tutorial_4.jpg)
+
+### Upload dataset
+
+Initially you need to create a manifest file for your image dataset. Information on how to do that is available
+on the [Simple command line to prepare dataset manifest file](/docs/manual/advanced/dataset_manifest) page.
```
Comment (nmanovic) [inline site/content/en/docs/manual/basics/attach-cloud-storage.md:57]: @TOsmanov , let's repeat here basic steps to upload a simple dataset. Need exact instructions. Let's take a video file (take a link from serverless tutorial), prepare a manifest and upload them.

Comment (TOsmanov) [reply site/content/en/docs/manual/basics/attach-cloud-storage.md:57]: @nmanovic , as far as I understand cloud storage into UI does not support the use of video file, do you want me to describe the division of the video into frame sequence?

Comment (nmanovic) [reply site/content/en/docs/manual/basics/attach-cloud-storage.md:57]: @Marishka17 , could you please comment on that?

Comment (nmanovic) [reply site/content/en/docs/manual/basics/attach-cloud-storage.md:57]: @TOsmanov , OK. Let's use a dataset with images in the case.

**Review by Marishka17 — COMMENTED** [review]
Great work!

**Review by Marishka17 — APPROVED** [review]

**Comment by Marishka17** [comment]
@TOsmanov , Could you please also prepare the same tutorial for Azure in another PR?

**Comment by TOsmanov** [comment]
> @TOsmanov , Could you please also prepare the same tutorial for Azure in another PR?

@Marishka17 Okay, I'll add

**Review by nmanovic — APPROVED** [review]
Great job!

**Pull request merged**